src/zino/statemodels.py

```diff
@@ -108,9 +108,9 @@ def is_juniper(self):
     def set_boot_time_from_uptime(self, uptime: int):
         """Calculates and sets the device boot time from a current uptime value.
 
-        :param uptime: An uptime value in 100ths of a second
+        :param uptime: An uptime value in seconds
         """
-        self.boot_time = now() - datetime.timedelta(seconds=uptime / 100)
+        self.boot_time = now() - datetime.timedelta(seconds=uptime)
```

Comment:
This moves the onus on the collector to convert from the centiseconds value collected from `sysUpTime`.  Why?